WorkshopToolCommon/DiscordWebhook.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Phoenix.WorkshopTool
+{
+    class DiscordWebhook
+    {
+        WorkshopType m_type;
+        string m_title;
+        string m_changelog;
+
+        //it is a small payload
+        static readonly string s_payload = @"{{""embeds"":[ {{""author"": {{""name"": ""{0}""}}, ""title"": ""{1}"", ""description"": ""{2}""}} ]}}";
```

Comment:
Static members are just prefixed with a single underscore (_), not s_.
Though, I appreciate the attention to the code style. 👍